rotkehlchen/tests/unit/decoders/test_velodrome.py

```diff
@@ -902,3 +902,289 @@ def test_get_reward_from_gauge_v2(optimism_accounts, optimism_transaction_decode
+    )
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=(timestamp := TimestampMS(1740997763000)),
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            amount=FVal(gas_str := '0.000000573869266137'),
+            location_label=user_address,
+            counterparty=CPT_GAS,
+            notes=f'Burn {gas_str} ETH for gas',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=19,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=Asset('eip155:10/erc721:0xFAf8FD17D9840595845582fCB047DF13f006787d/30064'),
+            amount=ZERO,
+            location_label=user_address,
+            address=string_to_evm_address('0xFAf8FD17D9840595845582fCB047DF13f006787d'),
+            extra_data={'token_id': 30064, 'lock_time': 1749686400},
+            counterparty=CPT_VELODROME,
+            notes='Increase unlock time to 12/06/2025',
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('optimism_accounts', [['0x21814F7481f07BA48d2CA224dBA62Bc1f5B447D0']])
+def test_claim_bribes(optimism_accounts, optimism_transaction_decoder):
+    user_address, evmhash = optimism_accounts[0], deserialize_evm_tx_hash('0xfc2df8e001236e5d4d2026f7d9943ef782d225af8676189f684347b1053776bc')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=optimism_transaction_decoder.evm_inquirer,
+        tx_hash=evmhash,
+    )
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=(timestamp := TimestampMS(1741027977000)),
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            amount=FVal(gas_str := '0.000001246550912592'),
+            location_label=user_address,
+            counterparty=CPT_GAS,
+            notes=f'Burn {gas_str} ETH for gas',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=143,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=Asset('eip155:10/erc20:0x747e42Eb0591547a0ab429B3627816208c734EA7'),
+            amount=FVal(receive_amount_1 := '0.088944623183973897'),
+            location_label=user_address,
+            address=string_to_evm_address('0x9015Af1A94d0c9896DB9BF359b62dc9B114d5587'),
+            counterparty=CPT_VELODROME,
+            notes=f'Claim {receive_amount_1} T from velodrome voter contract',
```

Comment:
In all these places, should be `a`. Also ... is it voter or bribe?

```suggestion
            notes=f'Claim {receive_amount_1} T from a velodrome voter contract',
```